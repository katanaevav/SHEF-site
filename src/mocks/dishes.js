export const Dishes = [
  {
    dishId: 1,
    dishTypeId: 1,
    dishCategory: 1,
    dishName: `Анатомия вкуса`,
    dishDescription: `Описание блюда Описание блюда
                      Описание блюда Описание блюда
                      Описание блюда Описание блюда
                      Описание блюда Описание блюда`,
    dishPrice: 250,
    dishCount: 1,
    dishWeight: `250 гр.`,
    dishTag: `NEW`,
    dishImage: `./img/dish.png`,
    dishImage2x: `./img/dish@2x.png`,
  },
  {
    dishId: 2,
    dishTypeId: 1,
    dishCategory: 1,
    dishName: `Солянка сборная мясная`,
    dishDescription: `Солянка сборная, приготовлена
                      из нескольких видов мяса. В состав
                      входят: телятина, свинина, курица,
                      копченые ребрышки, охотничьи колбаски,
                      свиной карбонад, копченая и вареная колбаса`,
    dishPrice: 210,
    dishCount: 1,
    dishWeight: `350 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/soup1.jpg`,
    dishImage2x: `./img/soup1.jpg`,
  },
  {
    dishId: 3,
    dishTypeId: 1,
    dishCategory: 1,
    dishName: `Картофельный суп с фрикадельками`,
    dishDescription: `Картофельный суп с фрикадельками и
                      вермишелью - один из самых легких
                      и быстрых мясных супов. `,
    dishPrice: 250,
    dishCount: 1,
    dishWeight: `250 гр.`,
    dishTag: ``,
    dishImage: `./img/soup2.jpg`,
    dishImage2x: `./img/soup2.jpg`,
  },
  {
    dishId: 4,
    dishTypeId: 1,
    dishName: `Борщ`,
    dishCategory: 1,
    dishDescription: `Борщ — разновидность супа на основе
                      свёклы, которая придаёт ему характерный
                      красный цвет.`,
    dishPrice: 300,
    dishCount: 1,
    dishWeight: `290 гр.`,
    dishTag: ``,
    dishImage: `./img/soup3.jpg`,
    dishImage2x: `./img/soup3.jpg`,
  },
  {
    dishId: 5,
    dishTypeId: 1,
    dishCategory: 1,
    dishName: `Чечевичный суп с мятой`,
    dishDescription: `Без чечевичного супа не обходится в Турции
                      ни один обед, а кое-где и завтрак. Главные
                      его ингредиенты — вода, красная чечевица и
                      перец, обычно тоже красный.`,
    dishPrice: 270,
    dishCount: 1,
    dishWeight: `220 гр.`,
    dishTag: ``,
    dishImage: `./img/soup4.jpg`,
    dishImage2x: `./img/soup4.jpg`,
  },
  {
    dishId: 6,
    dishTypeId: 2,
    dishCategory: 1,
    dishName: `Спагетти с соусом "А-ля Болоньезе"`,
    dishDescription: `Cпагетти с оригинальным соусом а-ля болоньезе.`,
    dishPrice: 270,
    dishCount: 1,
    dishWeight: `220 гр.`,
    dishTag: `%`,
    dishImage: `./img/vb1.jpg`,
    dishImage2x: `./img/vb1.jpg`,
  },
  {
    dishId: 7,
    dishTypeId: 2,
    dishCategory: 1,
    dishName: `Спагетти с баклажанами в томатном соусе`,
    dishDescription: `Быстрый и сытный обед из простых ингредиентов!
                      Паста с баклажанами в томатном соусе получается
                      очень вкусной и сытной.`,
    dishPrice: 280,
    dishCount: 1,
    dishWeight: `190 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/vb2.jpg`,
    dishImage2x: `./img/vb2.jpg`,
  },
  {
    dishId: 8,
    dishTypeId: 2,
    dishCategory: 1,
    dishName: `Паста с тунцом и помидорами`,
    dishDescription: `Прекрасный итальянский рецепт пасты с консервированным
                      тунцом в томатном соусе.`,
    dishPrice: 230,
    dishCount: 1,
    dishWeight: `230 гр.`,
    dishTag: `NEW`,
    dishImage: `./img/vb3.jpg`,
    dishImage2x: `./img/vb3.jpg`,
  },
  {
    dishId: 9,
    dishTypeId: 2,
    dishCategory: 1,
    dishName: `Пельмени`,
    dishDescription: `Домашние пельмени, слепленные вручную, на вкусном тесте
                      с сочной начинкой из фарша.`,
    dishPrice: 290,
    dishCount: 1,
    dishWeight: `250 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/vb4.jpg`,
    dishImage2x: `./img/vb4.jpg`,
  },
  {
    dishId: 10,
    dishTypeId: 2,
    dishCategory: 1,
    dishName: `Сосиска в Картофельном Тесте`,
    dishDescription: `Хот Дог без муки с картошкой и сыром. Быстро, просто и
                      очень вкусно!`,
    dishPrice: 290,
    dishCount: 1,
    dishWeight: `250 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/vb5.jpg`,
    dishImage2x: `./img/vb5.jpg`,
  },
  {
    dishId: 11,
    dishTypeId: 3,
    dishCategory: 2,
    dishName: `Рис`,
    dishDescription: `Считается основным (национальным) продуктом питания
                      в странах Юго-Восточной Азии и Китае`,
    dishPrice: 90,
    dishCount: 1,
    dishWeight: `200 гр.`,
    dishTag: ``,
    dishImage: `./img/g1.jpg`,
    dishImage2x: `./img/g1.jpg`,
  },
  {
    dishId: 12,
    dishTypeId: 3,
    dishCategory: 2,
    dishName: `Гречка`,
    dishDescription: `Каша, приготавливаемая из гречневой крупы, популярное
                      блюдо русской, белорусской, украинской, литовской и
                      польской кухонь.`,
    dishPrice: 90,
    dishCount: 1,
    dishWeight: `200 гр.`,
    dishTag: ``,
    dishImage: `./img/g2.jpg`,
    dishImage2x: `./img/g2.jpg`,
  },
  {
    dishId: 13,
    dishTypeId: 3,
    dishCategory: 2,
    dishName: `Картофель отварной`,
    dishDescription: `Один из самых распространенных и простых гарниров.`,
    dishPrice: 110,
    dishCount: 1,
    dishWeight: `200 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/g3.jpg`,
    dishImage2x: `./img/g3.jpg`,
  },
  {
    dishId: 14,
    dishTypeId: 1,
    dishCategory: 3,
    dishName: `Анатомия вкуса`,
    dishDescription: `Описание блюда Описание блюда
                      Описание блюда Описание блюда
                      Описание блюда Описание блюда
                      Описание блюда Описание блюда`,
    dishPrice: 250,
    dishCount: 1,
    dishWeight: `250 гр.`,
    dishTag: `NEW`,
    dishImage: `./img/dish.png`,
    dishImage2x: `./img/dish@2x.png`,
  },
  {
    dishId: 15,
    dishTypeId: 1,
    dishCategory: 3,
    dishName: `Солянка сборная мясная`,
    dishDescription: `Солянка сборная, приготовлена
                      из нескольких видов мяса. В состав
                      входят: телятина, свинина, курица,
                      копченые ребрышки, охотничьи колбаски,
                      свиной карбонад, копченая и вареная колбаса`,
    dishPrice: 210,
    dishCount: 1,
    dishWeight: `350 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/soup1.jpg`,
    dishImage2x: `./img/soup1.jpg`,
  },
  {
    dishId: 16,
    dishTypeId: 3,
    dishCategory: 3,
    dishName: `Рис`,
    dishDescription: `Считается основным (национальным) продуктом питания
                      в странах Юго-Восточной Азии и Китае`,
    dishPrice: 90,
    dishCount: 1,
    dishWeight: `200 гр.`,
    dishTag: ``,
    dishImage: `./img/g1.jpg`,
    dishImage2x: `./img/g1.jpg`,
  },
  {
    dishId: 17,
    dishTypeId: 3,
    dishCategory: 3,
    dishName: `Гречка`,
    dishDescription: `Каша, приготавливаемая из гречневой крупы, популярное
                      блюдо русской, белорусской, украинской, литовской и
                      польской кухонь.`,
    dishPrice: 90,
    dishCount: 1,
    dishWeight: `200 гр.`,
    dishTag: ``,
    dishImage: `./img/g2.jpg`,
    dishImage2x: `./img/g2.jpg`,
  },
];


export const CartDishes = [
  {
    dishId: 1,
    dishTypeId: 1,
    dishName: `Анатомия вкуса`,
    dishDescription: `Описание блюда Описание блюда
                      Описание блюда Описание блюда
                      Описание блюда Описание блюда
                      Описание блюда Описание блюда`,
    dishPrice: `250 р.`,
    dishWeight: `250 гр.`,
    dishTag: `NEW`,
    dishImage: `./img/dish.png`,
    dishImage2x: `./img/dish@2x.png`,
    dishCount: 3,
  },
  {
    dishId: 2,
    dishTypeId: 1,
    dishName: `Солянка сборная мясная`,
    dishDescription: `Солянка сборная, приготовлена
                      из нескольких видов мяса. В состав
                      входят: телятина, свинина, курица,
                      копченые ребрышки, охотничьи колбаски,
                      свиной карбонад, копченая и вареная колбаса`,
    dishPrice: `210 р.`,
    dishWeight: `350 гр.`,
    dishTag: `HOT`,
    dishImage: `./img/soup1.jpg`,
    dishImage2x: `./img/soup1.jpg`,
    dishCount: 1,
  },
  {
    dishId: 3,
    dishTypeId: 1,
    dishName: `Картофельный суп с фрикадельками`,
    dishDescription: `Картофельный суп с фрикадельками и
                      вермишелью - один из самых легких
                      и быстрых мясных супов. `,
    dishPrice: `250 р.`,
    dishWeight: `250 гр.`,
    dishTag: ``,
    dishImage: `./img/soup2.jpg`,
    dishImage2x: `./img/soup2.jpg`,
    dishCount: 2,
  },
];

