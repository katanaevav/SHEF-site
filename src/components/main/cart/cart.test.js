import React from "react";
import {configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Cart from "./cart.jsx";

import {Dishes} from "../../../mocks/dishes.js";

configure({adapter: new Adapter()});

it('Render Cart', () => {
  const tree = shallow(
    <Cart
      cartType = {1}
      cartDishesList = {Dishes}
    />
  );
  expect(tree).toMatchSnapshot();
});
