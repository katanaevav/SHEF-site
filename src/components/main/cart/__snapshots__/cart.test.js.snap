// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Cart 1`] = `ShallowWrapper {}`;
